### PR TITLE
Fix/double dom

### PR DESCRIPTION
**Pull request by tanjimhasan**

### CODE REVIEW

**Comment by kowsar89** [comment]
### Issue Details
Search Home
Double Dom Element after ajax success
https://prnt.sc/FklXA8MNOH9J

File: templates/search-form-contents.php [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <?php
+
```
Comment (kowsar89) [inline templates/search-form-contents.php:2]: unnecessary spacing change

File: templates/search-form-contents.php [hunk 2]
```diff
@@ -7,40 +8,44 @@
 
 use \Directorist\Helper;
 
-if ( ! defined( 'ABSPATH' ) ) exit;
+if (!defined('ABSPATH')) exit;
```
Comment (kowsar89) [inline templates/search-form-contents.php:11]: unnecessary spacing change

**Review by kowsar89 — CHANGES_REQUESTED** [review]
spacing updated in many places which is unnecessary

File: templates/search-form-contents.php (hunk 2 above)
Comment (tanjimhasan) [reply templates/search-form-contents.php]: Space removed

**Review by kowsar89 — APPROVED** [review]
works fine

**Pull request merged**